### DISCUSSION TITLE
TechnoTim just released a video about this repository :smiley:

### DISCUSSION THREAD

**Discussion by Mellowlynx** [tteck/Proxmox#3095] [General]
[Watch it on YouTube here](https://youtu.be/kcpu4z5eSEU?si=KIxS4QsoOMv8f85_)

**Comment by tesla-76** [comment]
This site deserves visibility for its efforts and because it helps even less experienced people to approach the world of virtualization and selfhosting. Long life and prosperity to Proxmox VE Helper-Scripts! 🖖🏻

**Reply by Mellowlynx** [reply]
Can't agree more 😃

**Comment by da99Beast** [comment]
I am here because of that TechnoTim video.  This is an awesome collection of scripts and I thank all those who have contributed to the project - both the actual scripts and the ones in the discussion area!
